vetomint/tests/unit_action.rs

```diff
@@ -0,0 +1,150 @@
+use vetomint::*;
+
+#[allow(dead_code)]
+pub fn get_response_from_event(
```

Comment:
Let's discuss about what layer of abstraction is needed for those auxiliary functions